tests/unit/processing_container/test_process.py

```diff
@@ -51,23 +52,42 @@ def clean_generated_files(cls):
             if os.path.exists(f):
                 os.remove(f)
 
-    @patch("backend.corpora.dataset_processing.process.download_from_source_uri")
-    def test_main(self, mock_download_from_source_uri):
+    @patch("backend.corpora.dataset_processing.process_download_validate.download_from_source_uri")
+    @patch("backend.corpora.dataset_processing.remaster_cxg.process")  # TODO: provide test data to properly test this.
+    def test_main(self, mock_rematser_cxg_process, mock_download_from_source_uri):
```

Comment:
Updated to use `process.main` instead of `process.process`. This will give us better and more realistic testing conditions.